apps/labs/posts/cupy-czi-grant-year1.md

```diff
@@ -0,0 +1,121 @@
+
+<figure>
+  <img src="/posts/cupy-czi-grant-year1/filtered_signals.png" alt="A grid of four plots, arranged in a 2x2 configuration. Each plot shows the result of taking a noisy signal, applying the lfilter and filtfilt CuPy in the following order: lfilter once, lfilter twice and filfilt. The filtfilt results reconstruct the original signal with higher fidelity." />
+  <figcaption style={{textAlign: "center"}}>Filtering results using the new lfilter and filtfilt CuPy APIs. Each result is compared against the original, non-noisy signal.</figcaption>
+</figure>
+
+As part of this effort, we made several other improvements to other CuPy modules, such as adding special functions for elliptic filters, implementing the matrix exponential in the `cupyx.scipy.linalg` namespace, and implementing KD-Trees in `cupyx.scipy.spatial`.
+
+This work would not be possible without the help of the [cuSignal](https://github.com/rapidsai/cusignal) team at the NVIDIA RAPIDS initiative, where more than 36 APIs will be migrated from their project into CuPy, thus unifying the GPU signal processing efforts currently available in the ecosystem. As part of this migration, cuSignal will be deprecated and replaced by the new `cupyx.scipy.signal` module calls. The calls will be interchangeable by simply replacing the `cusignal` imports with imports from `cupyx.scipy.signal`. For more information about this collaboration, we invite you to visit the [RAPIDS blog](https://medium.com/rapids-ai).
+
+As the grant enters its second year, we plan to finish the overall interoperability of the interpolation module, which first requires a significant refactor of several FORTRAN routines for spline smoothing into GPU kernels and, second, implementing computational geometry routines to compute convex hulls and Delaunay triangulations, which are used for interpolating multi-dimensional unstructured data. This work will further enhance overall CuPy-SciPy compatibility.
+
+These new APIs will open a sea of possibilities in terms of acceleration and batch processing of data, and we hope they will be useful to the CuPy community. Right now, these additions are being pre-released as part of the beta releases of the upcoming CuPy v13.0. We invite you to keep an eye on our tracking issues https://github.com/cupy/cupy/issues/7186 and https://github.com/cupy/cupy/issues/7403, where you can find more information regarding the implementation discussions, as well as the current progress on the remaining functionality of CuPy’s interpolation and signal modules. Please feel free to chime in and contribute to any discussion or improvement to the project!
+
+We would like to thank the [Chan Zuckerberg Initiative (CZI)](https://chanzuckerberg.com/eoss/), Preferred Networks, and Quansight Labs for the resources used to plan and execute the current grant. We would also like to thank NVIDIA and the RAPIDS team for their help and willingness to migrate cuSignal into CuPy as part of this work.
```

Comment:
This looks slightly off with CZI being a link and Preferred Networks and following entities not. I'd suggest making them all links, or none.

Also, this is the Labs blog, so no need to thank ourselves here I'd say? The mention in the next paragraph is probably enough.